authors.yml

```
@@ -8,4 +8,10 @@ posdealer:
   name: PosDealer Experience Team
   title: The team working on the Portal and other services for PosDealers
   email: feedback+portal@fiskaltrust.cloud
-  image_url: https://avatars.githubusercontent.com/u/19596496?s=200&v=4
\ No newline at end of file
+  image_url: https://avatars.githubusercontent.com/u/19596496?s=200&v=4
+  
+digital_receipt:
+  name: Digital Receipt Team
+  title: The team working on the digital receipt
+  email: feedback+middleware@fiskaltrust.cloud
```

```suggestion
  email: feedback+digitalreceipt@fiskaltrust.cloud
```

All those emails are anyway caught in the same inbox :)